lib/matplotlib/figure.py

```diff
@@ -1838,21 +1838,25 @@ def subplot_mosaic(self, mosaic, *, sharex=False, sharey=False,
             Defines the relative widths of the columns. Each column gets a
             relative width of ``width_ratios[i] / sum(width_ratios)``.
             If not given, all columns will have the same width.  Equivalent
-            to ``gridspec_kw={'width_ratios': [...]}``.
+            to ``gridspec_kw={'width_ratios': [...]}``. This argument applies
+            only to the outer layout.
```

Comment:
```suggestion
            to ``gridspec_kw={'width_ratios': [...]}``. In the case of nested
            layouts, this argument applies only to the outer layout.
```
We should give the "nested context" because that's a rather advanced case and the whole docstring does not mention nested layouts or "outer" (that's only in the corresponding tutorial).

Reply:
Ah yeah I hadn't noticed that. Good point, I'll commit these in